server/app/views/admin/migration/AdminImportViewPartial.java

```diff
@@ -40,21 +47,39 @@ public DomContent renderError(String errorMessage) {
   }
 
   /** Renders the correctly parsed program data. */
-  public DomContent renderProgramData(ProgramMigrationWrapper programMigrationWrapper) {
+  public DomContent renderProgramData(
+      Http.Request request, ProgramMigrationWrapper programMigrationWrapper, String json) {
     ProgramDefinition program = programMigrationWrapper.getProgram();
     DivTag programDiv =
         div()
             .withId(PROGRAM_DATA_ID)
             .with(
                 h3("Program name: " + program.localizedName().getDefault()),
-                p("Admin name: " + program.adminName()));
+                h4("Admin name: " + program.adminName()));
```

Comment:
The test locator couldn't find this when it was a ptag so I changed it to an `h4`. Maybe related to this bug: https://github.com/microsoft/playwright/issues/23377